### DISCUSSION TITLE
Most important upgrade to remain relevent

### DISCUSSION THREAD

**Discussion by mikechy** [MasaCMS/MasaCMS#187] [Ideas]
At this point, I think the most important addition to this amazing CMS is a true page builder that can compete with the WordPress offerings. We've had at least a dozen new projects in the past six months go WordPress because of the power of the page builders. We've demoed Masa to all of them and in he end, it just can't compete with Elementor, Beaver Builder or Breakdance.  

Is there any plan in the works to complete at this level?  It would be a game-changer.

**Comment by Tigerlady** [comment]
I agree; I have lost good deals because of the ease of use for WordPress.  I always have to hack away at stuff to get some of the new modern designs everyone wants to work on.  I do not work in PHP or WordPress, but it is always the easy way for clients to go.
